decentraland/helpers/index.js

```diff
@@ -107,16 +107,10 @@ export const fundAccountWithMana = accountAddress => {
   return action;
 };
 
-const getTransactionHashFromAction = async action => {
```

Comment:
We could consider keeping this helper around. But maybe it's bad practice to keep legacy unused helpers around, even if they might be helpful in the future.

Reply:
Even the `action.getId()` function doing the same as this function?

Reply:
Oh, I missed that. Nope, in that case I'm glad you removed this. Resolving this.